modules/swagger-codegen/src/main/java/com/wordnik/swagger/codegen/DefaultCodegen.java

```diff
@@ -904,6 +904,10 @@ public CodegenParameter fromParameter(Parameter param, Set<String> imports) {
       p.required = param.getRequired();
     p.jsonSchema = Json.pretty(param);
 
+    if(param instanceof QueryParameter) {
+        p.defaultValue = ((QueryParameter)param).getDefaultValue();
+    }
+
```

Comment:
@wing328 it's now moving over default values for forms and headers, as requested.  I updated the corresponding SpringMVC templates as well, since I am using those and could test them.  (Note: Spring RequestPart, which supports Form elements, does not support default values)